examples/tests/v3/test_00_consumer.py

```diff
@@ -75,24 +75,17 @@ def test_get_existing_user(pact: Pact) -> None:
     expected: Dict[str, Any] = {
         "id": 123,
         "name": "Verna Hampton",
-        "created_on": {
-            # This structure is using the Integration JSON format as described
-            # in the link below. The preview of V3 currently does not have
-            # built-in support for matchers and generators, though this is on
-            # the roadmap and will be available before the final release.
-            #
-            # <https://docs.pact.io/implementation_guides/rust/pact_ffi/integrationjson>
-            "pact:matcher:type": "regex",
-            "regex": r"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{6}(Z|(\+|-)\d{2}:\d{2})",
-            "value": datetime.now(tz=timezone.utc).isoformat(),
-        },
+        "created_on": match.datetime(
+            datetime.now(tz=timezone.utc).isoformat(),
+            format="%Y-%m-%dT%H:%M:%S.%fZ",
```

Comment:
If a Python-native `datetime` object is passed in, it is automatically formatted into the target format. I think this would provide a nicer user experience (and slightly more Pythonic).